static/enRefMerged/de.json

```diff
@@ -1474,10 +1474,10 @@
   "Mega Eggs": "Mega Eier",
   "Mega Legendary Egg": "Mega Legendäres Ei",
   "Mega Legendary Eggs": "Mega Legendäre Eier",
-  "Ultra Beast Egg": "Ultra Bestien Ei",
-  "Ultra Beast Eggs": "Ultra Bestien Eier",
-  "Elite Egg": "Elite Ei",
-  "Elite Eggs": "Elite Eier",
+  "Ultra Beast Egg": "Ultrapforte",
+  "Ultra Beast Eggs": "Ultrapforten",
```

Comment:
It's weird that the key contains still "egg", when you changed also english Translation.. other than that, good for me :)